packages/cross_file/example/pubspec.yaml

```
@@ -0,0 +1,25 @@
+name: cross_file_example
+description: Demonstrates how to use cross files.
+publish_to: none
+
+environment:
+  sdk: ">=2.18.0 <4.0.0"
+  flutter: ">=3.3.0"
+
+dependencies:
+  cross_file:
+    # When depending on this package from a real application you should use:
+    #   cross_file: ^x.y.z
+    # See https://dart.dev/tools/pub/dependencies#version-constraints
+    # The example app is bundled with the plugin so we use a path dependency on
+    # the parent directory to use the current plugin's version.
+    path: ../
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  async: ^2.5.0
+  flutter_test:
+    sdk: flutter
+  integration_test:
+    sdk: flutter
```

Adds empty line at the end of the file:

```suggestion
    sdk: flutter

```